clients/web/src/components/warnings/brave-warning.tsx

```diff
@@ -0,0 +1,52 @@
+import { JSXElement, Show, Suspense } from "solid-js";
+import { Info } from "../icons/info";
+
+// The Brave browser exports a navigator function to determine whether we are using it.
+declare global {
+  interface Navigator {
+    brave?: {
+      isBrave: () => Promise<boolean>;
+    };
+  }
+}
+
+export function BraveWarning() {
+  const isBrave = (navigator.brave && navigator.brave.isBrave()) || false;
```

Comment:
better use `const isBrave = !!navigator.brave?.isBrave();`

Reply:
Thanks for the feedback, will use this version :)